app/views/bikes/edit_theft_details.html.haml

```diff
@@ -39,13 +41,13 @@
               .form-well-input
                 = srecord.text_field :secondary_phone, class: 'form-control'
 
-            = render partial: '/bikes/bike_fields/stolen_location', locals: { srecord: srecord, stolen_type: stolen_type }
+            = render partial: 'bikes/bike_fields/stolen_location', locals: { srecord: srecord, stolen_type: stolen_type }
 
             .form-group.row.unnested-field
               = srecord.label :estimated_value, class: 'form-well-label'
               .form-well-input.input-group{ style: 'max-height: 2.5rem;'}
                 .input-group-addon
-                  $
+                  = t("number.currency.format.unit")
```

Comment:
If we're accepting things in non-usd format, we need to store the unit they're being submitted in :/

Reply:
I think it makes sense to punt on dealing with that for now, and just make everyone submit in USD for now

Reply:
Cool. Just a head's up that pricing plans are also not localized yet -- right now we're localizing the currency format but not the actual price (since I'm assuming we'll want to establish price points independently). Alternatively, we could just convert currency into local currency.

Reply:
It looks like Stripe will convert when we send the charge to them? 

We might want to just always display USD for now.